resources/js/components/OnboardingModal.vue

```diff
@@ -1,83 +1,54 @@
-            <div
-                class="text-base mb-4"
-                v-html="$t('Save interesting artworks from the exhibition to your mobile device.')"
-            ></div>
-            <div
-                class="text-base"
-                v-html="
-                    $t(
-                        'Insert the code located by the artwork into the grid. Press save in the artwork profile to collect it.'
-                    )
-                "
-            ></div>
-            <ConfirmButton class="bg-black text-white my-4" @click="next">{{ $t('Next') }}</ConfirmButton>
-        </OnboardingStep>
-
-        <OnboardingStep v-if="step == 1">
-            <h3 class="font-bold text-2xl my-4">{{ $t('Create a collection') }}</h3>
-            <div
-                class="text-base mb-4"
-                v-html="$t('Saved artworks are added into your collection. Return to them later to find out more.')"
-            ></div>
-            <div
-                class="text-base"
-                v-html="$t('The collection will persist even after you close the application.')"
-            ></div>
-            <ConfirmButton class="bg-black text-white my-4" @click="next">{{ $t('Next') }}</ConfirmButton>
-        </OnboardingStep>
-
-        <OnboardingStep v-if="step == 2">
-            <h3 class="font-bold text-2xl my-4">{{ $t('Share the collection') }}</h3>
-            <div
-                class="text-base mb-4"
-                v-html="
-                    $t(
-                        'Tap the Share button in your collection, and send the link to your e-mail or share it with friends.'
-                    )
-                "
-            ></div>
-            <div class="text-base font-bold">{{ $t('The shared link never expires.') }}</div>
-            <ConfirmButton
-                class="bg-black text-white my-4"
-                @click="
-                    reset(),
-                    emit('close')
-                "
-                >{{ $t('Start collecting') }}</ConfirmButton
-            >
-        </OnboardingStep>
+    <div class="md:max-w-lg md:mx-auto fixed inset-0 z-50 flex justify-center items-center" v-if="visible">
+        <div class="bg-black opacity-70 absolute inset-0 cursor-zoom-out" @click="close" />
+        <Carousel :items-to-show="2.1" :wrap-around="true" class="min-w-[calc(175%)]" ref="myCarousel">
+            <Slide :key="0">
+                <OnboardingStep
+                    :btnFn="next"
+                    :title="'Collect art'"
+                    :body1="'Collect art'"
+                    :body2="'Save interesting artworks from the exhibition to your mobile device.'"
+                    :btnText="'Next'"
+                    :active-step="1"
+                />
+            </Slide>
+            <Slide :key="1" class="w-screen">
+                <OnboardingStep
+                    :btnFn="next"
+                    :title="$t('Create a collection')"
```

Comment:
```suggestion
                    :title="'Create a collection'"
```
it's translated inside component
(to be honest, I don't know, whether it's better to translated inside component - or to keep child component "_as dumb as possible_" a do it here?